### PR TITLE
RHBA-608 - UnsupportedOperationException when starting a process with…

### PR DESCRIPTION
**Pull request by mswiderski**
… KieSpringTransactionManagerFactory

### CODE REVIEW

**Comment by mswiderski** [comment]
Jenkins retest this

File: kie-spring/src/main/java/org/kie/spring/persistence/KieSpringTransactionManagerFactory.java [hunk 1]
```diff
@@ -24,9 +24,14 @@
 
 public class KieSpringTransactionManagerFactory extends TransactionManagerFactory {
 
+    private AbstractPlatformTransactionManager globalTransactionManager;
+    
     @Override
     public TransactionManager newTransactionManager() {
-        throw new UnsupportedOperationException("use newTransactionManager(Environment) instead");
+        if (globalTransactionManager == null) {
+            throw new RuntimeException("No transaction manager set nor environemtn provided to loo it up");
```
Comment (sutaakar) [inline kie-spring/src/main/java/org/kie/spring/persistence/KieSpringTransactionManagerFactory.java]: A typo: environemtn -> environment, loo -> look

Comment (mswiderski) [reply kie-spring/src/main/java/org/kie/spring/persistence/KieSpringTransactionManagerFactory.java]: typo fixed

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Pull request merged**